### ISSUE TITLE
Does Splash support POST method?

### ISSUE THREAD

**Issue by microhello** [scrapinghub/splash#239]
@kmike 
I want to fetch some webpages through POST method using splash http api, but seem like splash only support GET method: 
```python

    def _send_request(self, url, callback, method='GET', body=None,
                      headers=None):
        # XXX: The caller must ensure self._delete_reply is called in a callback.
        if method != 'GET':
            raise NotImplementedError()
```
https://github.com/scrapinghub/splash/blob/master/splash/browser_tab.py#L808-809 .

If not , how to do?
Thanks a lot.

**Comment by kmike** [comment]
You can make an AJAX request using splash:runjs. 
Or help adding POST method to Splash :) It was not implemented because of lack of time, not because it is complicated.

**Comment by pawelmhm** [comment]
:+1:  I need this feature for one spider

**Comment by pawelmhm** [comment]
So our TODO would look like this:

#### lua scripting
- [ ] update splash:go to support passing POST method as one of arguments and passing formbody
- [ ] perhaps add http_post along http_get splash method? 

#### HTTP API
- [ ] update HTTP API to accept method argument and formbody argument for outcoming splash requests and pass those arguments to splash

should we split this into couple of issues?

**Comment by kmike** [comment]
Fixed by #276.
